src/Castle.Core.Tests/Castle.Core.Tests-VS2017.csproj

```diff
@@ -30,4 +33,9 @@
     <Reference Include="System.Windows.Forms" />
     <Reference Include="WindowsBase" />
   </ItemGroup>
+  <ItemGroup>
+    <None Update="log4netIntegration\log4net.xml">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
This comes again from tests running with their working directory in bin rather than the source directory, that's fine though.